test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1424,86 +1566,82 @@ def test_cgroup_job_array(self):
         self.assertFalse(self.is_dir(cpath, ehost1))
-
         # Verify if subjob2 is running
         subj2 = jid.replace('[]', '[2]')
         self.server.expect(JOB, {'job_state': 'R'}, id=subj2)
         # Force delete the subjob and verify cgroups
         # files are cleaned up
-        self.server.delete(id=subj2, extend="force")
+        self.server.delete(id=subj2, extend='force')
         self.server.expect(JOB, {'job_state': 'X'}, subj2)
         # Adding extra sleep for file to clean up
         time.sleep(2)
-        cpath = os.path.join(self.paths['memory'], 'pbspro', subj2)
-        self.assertFalse(self.is_dir(cpath, self.momA))
+        cpath = self.get_cgroup_job_dir('memory', subj2, ehost1)
+        self.assertFalse(self.is_dir(cpath, ehost1))
 
     def test_cgroup_cleanup(self):
         """
         Test that cgroups files are cleaned up after qdel
         """
         # Test requires 2 nodes
         if len(self.moms) != 2:
-            self.skipTest("Test requires two Moms as input, " +
-                          "use -p moms=<mom1:mom2>")
-
+            self.skipTest('Test requires two Moms as input, '
+                          'use -p moms=<mom1:mom2>')
         hook_body = """
 import pbs
 import time
 time.sleep(20)
 """
-        a = {'event': "execjob_prologue", "enabled": "True"}
-        self.server.create_import_hook("pro", a, hook_body)
-        self.load_config(self.cfg1 % ("", "", "", "", self.swapctl))
-        a = {'Resource_List.select': '2:ncpus=1:mem=300mb',
+        a = {'event': 'execjob_prologue', 'enabled': 'True'}
+        self.server.create_import_hook('pro', a, hook_body)
+        self.load_config(self.cfg1 % ('', '', '', '', self.swapctl))
+        a = {'Resource_List.select': '2:ncpus=1:mem=100mb',
              'Resource_List.place': 'scatter'}
         j = Job(TEST_USER, attrs=a)
         j.create_script(self.sleep15_job)
         jid = self.server.submit(j)
         a = {'substate': '41'}
         self.server.expect(JOB, a, jid)
-
         self.server.status(JOB, ['exec_host'], jid)
         ehost = j.attributes['exec_host']
         tmp_host = ehost.split('+')
         ehost1 = tmp_host[0].split('/')[0]
         ehost2 = tmp_host[1].split('/')[0]
         # Adding delay for cgroups files to get created
         time.sleep(1)
-        cpath = os.path.join(self.paths['cpuset'], 'pbspro', jid)
-        self.assertTrue(self.is_dir(cpath, ehost1))
-        self.assertTrue(self.is_dir(cpath, ehost2))
-
+        ehjd1 = self.get_cgroup_job_dir('cpuset', jid, ehost1)
+        self.assertTrue(self.is_dir(ehjd1, ehost1))
+        ehjd2 = self.get_cgroup_job_dir('cpuset', jid, ehost2)
+        self.assertTrue(self.is_dir(ehjd2, ehost2))
         self.server.delete(id=jid, wait=True)
-
-        self.assertFalse(self.is_dir(cpath, ehost1))
-        self.assertFalse(self.is_dir(cpath, ehost2))
+        self.assertFalse(self.is_dir(ehjd1, ehost1))
+        self.assertFalse(self.is_dir(ehjd2, ehost2))
```

Comment:
Consider adding a message to the above 3 asserts

Reply:
Done.